config/spryk/spryks/Spryker/AddModulePayment.yml

```diff
@@ -0,0 +1,40 @@
+spryk: structure
+description: "Adds a new payment module structure."
+priority: 1
+mode: both
+level: 1
+
+arguments:
+    organization:
+        default: SprykerEco
```

Comment:
If I don't specify module name 
`spryk:run AddModulePayment Test` will it ask me to provide organization? what does it mean default in this case?

Reply:
Yes, it will ask you anyway. Default value will be used only if you will not choose variant